trigger/task.go

```diff
@@ -355,7 +355,10 @@ func (t *taskTriggers) generate(sub *event.Subscription, pastTenseOverride, test
 					return nil, errors.Wrapf(err, "error getting execution task")
 				}
 				if executionTask.Details.Status == evergreen.TaskFailed {
-					if executionTask.Details.Description == evergreen.TaskDescriptionStranded {
+					skipTicket := executionTask.Details.Description == evergreen.TaskDescriptionStranded ||
```

Comment:
can set shouldSkipTicket here (instead of creating an equivalent `skipTicket` variable) and then just break if shouldSkipTicket is false

Reply:
Oh yeah that is much cleaner, thanks!